app/components/channel_drawer/channel_list.js

```diff
@@ -287,6 +399,17 @@ class ChannelList extends React.Component {
         return data;
     };
 
+    modalOnChange(option) {
+        this.props.handleDisableDrawer(false);
+        if (option) {
+            option.action();
+        } else {
+            this.setState({
```

Comment:
Ok I'm going to leave this one as it is cause in the event that the `leave channel` is selected which shows later on an alert, there is a weird behavior that makes the alert shown in the background but blocking the screen which makes the app unusable, this way its handled correctly, I'll add a comment in the code